xcheckmypkgs

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+for pkg in $('/usr/bin/xmypkgs'); do
```

Comment:
removed single quotes.




Diogo Leal [estranho]
Site : http://diogoleal.com
talk: estranho@jabber-br.org

On Sun, Jul 10, 2016 at 1:47 AM, Toyam Cox <notifications@github.com> wrote:

> In xcheckmypkgs
> <https://github.com/chneukirchen/xtools/pull/45#discussion_r70175853>:
>
> > @@ -0,0 +1,6 @@
> > +#!/bin/sh
> > +
> > +for pkg in $('/usr/bin/xmypkgs'); do
>
> No single quotes necessary here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chneukirchen/xtools/pull/45/files/481ecfcb64f71517ab95f5db8588f3dcac889171#r70175853>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AASN_e5eJKfNuPDtEuALfq3P7xMDELl1ks5qUHlggaJpZM4JIpTI>
> .
>